### PR TITLE
[Remove Stream/Optional usage] LPS-171937 translation-web

### PR DESCRIPTION
**Pull request by haydenzheng314**

### CODE REVIEW

File: modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/asset/model/TranslationEntryAssetRenderer.java [hunk 1]
```diff
@@ -109,15 +108,14 @@ public String getTitle(Locale locale) {
...
-				_translationEntry.getClassPK(), locale);
+		String infoItemTitle = infoItemHelper.getInfoItemTitle(
+			_translationEntry.getClassPK(), locale);
 
 		return LanguageUtil.format(
 			locale, "translation-of-x-to-x",
 			new Object[] {
-				infoItemTitleOptional.orElseGet(
-					() -> _getAssetRendererTitle(locale)),
+				(infoItemTitle != null) ? infoItemTitle :
```
Comment (dantewang) [inline modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/asset/model/TranslationEntryAssetRenderer.java:117]: @LisztLiszt It seems we rarely use `A ? B : C` in Java code. Please do 

```
if (infoItemTitle == null) {
    infoItemTitle = _getAssetRendererTitle(locale);
}
```

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by LisztLiszt** [comment]
ci:test:sf

**Comment by LisztLiszt** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:forward

cc @shuyangzhou @tinatian

**Pull request closed without merging**